quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImpl.java

```
@@ -43,7 +43,11 @@ public Collection addCollection(String name, String description) {
      **/
     @Override
     public Collection findCollection(String id) {
-        return collectionRepo.findById(new ObjectId(id)).orElse(null);
+        try {
+            return collectionRepo.findById(new ObjectId(id)).orElse(null);
+        } catch(IllegalArgumentException e) {
```

don't swallow the exception, it should be logged.